README.md

```diff
@@ -1,5 +1,12 @@
 ## Fyyur
 
+### Dependencies
+This project depends on the following
+1. Flask
+2. Babel
+3. Sql-Alchemy
+4. Psycopg2
```

Comment:
@pallavika-nandamuri Can you also add the project links to these dependencies?

Reply:
Added the dependencies links aswell

Reply:
Thanks